Hazel/src/Hazel/Scene/SceneSerializer.cpp

```diff
@@ -239,6 +270,10 @@ namespace Hazel {
 				{
 					auto& src = deserializedEntity.AddComponent<SpriteRendererComponent>();
 					src.Color = spriteRendererComponent["Color"].as<glm::vec4>();
+
+					auto uvoff_node = spriteRendererComponent["TextureUVOffset"];
+					if(uvoff_node)
+						src.TextureUVOffset = spriteRendererComponent["TextureUVOffset"].as<glm::vec2>();
```

Comment:
No need to check if its valid, it will always be serialized, it will only error on existing saved scenes, this PR should update those as well, rather than checking, for sure since we don't have a released version yet.
```suggestion
					src.TextureUVOffset = spriteRendererComponent["TextureUVOffset"].as<glm::vec2>();
```